sklearnex/_utils.py

```diff
@@ -155,6 +156,30 @@ def _doc_link_template(self) -> str:
         return f"https://uxlfoundation.github.io/scikit-learn-intelex/latest/non-scikit-algorithms.html#{module_path}.{class_name}"
 
 
+# Adds a small note note about serialization for extension estimators that are incremental.
+# The class docstrings should leave a placeholder '%incremental_serialization_note%' inside
+# their docstrings, which will be replaced by this note.
+def _add_inc_serialization_note(class_docstrings: str) -> str:

```

Comment:
Docstring gives better readability:
```suggestion
def _add_inc_serialization_note(class_docstrings: str) -> str:
    """Adds a small note note about serialization for extension estimators that are incremental.
    The class docstrings should leave a placeholder '%incremental_serialization_note%' inside
    their docstrings, which will be replaced by this note.
    """
```